libmscore/style.cpp

```diff
@@ -305,12 +305,12 @@ static const StyleType styleTypes[] {
       { Sid::fretNumPos,              "fretNumPos",              QVariant(0) },
       { Sid::fretY,                   "fretY",                   Spatium(2.0) },
       { Sid::fretMinDistance,         "fretMinDistance",         Spatium(0.5) },
-      { Sid::fretMag,                 "fretMag",                 QVariant(1.0) },
+      { Sid::fretMag,                 "fretMag",                 QVariant(1) },
```

Comment:
Since you're not changing the value after all, it would be better not to change this line at all.

Reply:
Indeed, just an artifact from when I changed the default. I'll change it back.